packages/react/src/providers/core-provider/index.tsx

```diff
@@ -63,14 +73,25 @@ export const CarrotCoreProvider = ({
     providers,
     getConnectors,
     ipfsGatewayURL,
+    reactQueryClient,
 }: CarrotCoreProviderProps) => {
+    persistQueryClient({
+        queryClient: reactQueryClient,
+        persister: createSyncStoragePersister({
+            key: "carrot-kpi-react-query-cache",
+            storage: window.localStorage,
+        }),
+    });
+
     if (!!ipfsGatewayURL) IPFSService.gateway = ipfsGatewayURL;
+
     return (
         <PreferencesProvider>
             <WagmiSetup
```

Comment:
doesn't make sense to me to pass the react query client as props to `WagmiSetup` when I read this I think they are related.

Can we just put the react query bellow `PreferencesProvider`?

Reply:
Ah sheesh, now that you say it you're right. Fixing